diff --git a/lib/src/data/fooddb_repository.dart b/lib/src/data/fooddb_repository.dart
index d4daa08..cf26fd8 100644
--- a/lib/src/data/fooddb_repository.dart
+++ b/lib/src/data/fooddb_repository.dart
@@ -26,6 +26,7 @@ class FoodDbRepository {
     int page = 1,
     String? categoryEn,
     String? brandEn,
+    String? countryEn,
   }) async {
     try {
       return await _search.search(
@@ -34,6 +35,7 @@ class FoodDbRepository {
         pageSize: 20,
         categoryEn: categoryEn,
         brandEn: brandEn,
+        countryEn: countryEn,
       );
     } catch (_) {
       return <Product>[];
